bindings/python/crocoddyl/multibody/actuations/multicopter-base.cpp

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh, IRI: CSIC-UPC
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/multibody/multibody.hpp"
+#include "crocoddyl/multibody/actuations/multicopter-base.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeActuationModelMCBase() {
```

Comment:
Can we please spell "MC" out to "MultiCopter"?

Reply:
Sure, will do